README.md

```diff
@@ -426,6 +421,11 @@ export default {
 };
 ```
 
+To run tests which belong to that group:
+```bash
+d2l-test-runner --group safari-only
```

Comment:
maybe `npx d2l-test-runner --group safari-only` since you can't run it directly?

Reply:
I've reverted this change out anyway since we're just going to support this, but I think I'll leave it without the `npx` as there are a bunch of other places in the README which exclude it as well. The `npx` isn't needed if it's being run from a script in `package.json`, so I think it's best to leave it up to the developer to recognize the correct way to access to binary.